scripts/run.sh

```diff
@@ -1,14 +1,15 @@
 #!/bin/bash
 
-# Debugging
-#echo '/tmp/core.%e.%p.%t' | sudo tee /proc/sys/kernel/core_pattern
-#ulimit -c unlimited
-
 # Add -debug to see engine output
-# Missing -repeat and -openings (-n gives some randomness)
-DIR=$PWD/../src
-./cutechess-cli -rounds 100 -tournament gauntlet -concurrency 1 \
+WDR=$HOME/Workspace/chess/lczero-weights
+DIR=$PWD/../build
+NXT=lc_gen2
+CUR=lc_gen1
+
+cutechess-cli -rounds 50 -tournament gauntlet -concurrency 4 \
  -pgnout results.pgn \
- -engine name=lc_new cmd=$DIR/lczero arg="-n" arg="-t1" arg="--weights=$DIR/newweights.txt" \
- -engine name=lc_base cmd=$DIR/lczero arg="-n" arg="-t1" arg="--weights=$DIR/weights.txt" \
- -each proto=uci tc=inf
+ -engine name=$NXT cmd=$DIR/lczero arg="--threads=1" arg="--noise" arg="--weights=$WDR/gen2-64x6.txt" arg="--playouts=800" arg="--noponder" arg="--gpu=0" tc=inf \
+ -engine name=$CUR cmd=$DIR/lczero arg="--threads=1" arg="--noise" arg="--weights=$WDR/seed-64x6.txt" arg="--playouts=800" arg="--noponder" arg="--gpu=1" tc=inf \
```

Comment:
Super small thing - why not leave the "tc=inf" in the "each" below?

Reply:
Oh that's a remminent from playing gnuchess. I'll fix it when at my computer again.